### ISSUE TITLE
Need method to manage spots on device

### ISSUE THREAD

**Issue by zmichels** [StraboSpot/strabo-mobile#174]
The current spot management system is really rough to navigate and edit, and I currently don't know how to do what I want and keep the new data collected in my device. I am wanting to remove all the spots from certain datasets I downloaded (remove option on sync isn't removing and just deletes from server)... But I don't want to delete all the spots on device because I have spots collected in the field that don't work to upload (see issue about upload sync stalling). As such... And in general, we need an easy way to edit the spots on the device... Maybe to start... a list with selectable buttons next to spots (similar to the interface when picking spots to include in a group), and then actions to delete or upload.
I have nearly 700 spots on here now. I can't fathom deleting each one from within the entry.

**Comment by jessica-azgs** [comment]
@zmichels Yes, this has been an issue I've wanted to clean up because I totally agree with you. There have just been so many other issues to deal with and you're the first one to add so many spots in the app. But yes, it's on the radar! Thanks for all the testing and posting of issues! It really helps.

**Comment by zmichels** [comment]
:-)
I'm glad it helps. I'm sorry it's generally late-to-the-game and in bursts as I come in and out of the field. I really appreciate all the help! I don't mean to ignore the many good aspects of the app either. I'm pretty amazed at everything that it already allows me to do, really.

**Comment by zmichels** [comment]
What about a way to at least toggle the display on/off for different groups/datasets/spots?
